apps/web/src/services/workspace-home/components/BookmarkBoard.vue

```diff
@@ -81,23 +78,24 @@ const state = reactive({
         }
         return defaultSets;
     }),
-    menuVisible: false,
+    visibleMenu: false,
```

Comment:
nitpick: how about isMenuVisible?

Reply:
I'm not using this state, so I'll delete it!